From 2fe8ea93d8fbba470ec3ea20cdc6f9083b86b0bc Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Thu, 19 Sep 2024 09:19:56 +0200
Subject: [PATCH] Streamline build with BUILD_WITH_BLASFEO=OFF

---
 external/CMakeLists.txt | 14 +-------------
 1 file changed, 1 insertion(+), 13 deletions(-)

diff --git a/external/CMakeLists.txt b/external/CMakeLists.txt
index 8aed9c5..c1d8e8c 100644
--- a/external/CMakeLists.txt
+++ b/external/CMakeLists.txt
@@ -30,19 +30,7 @@ if (BUILD_WITH_BLASFEO)
     add_subdirectory(blasfeo)
 
 else()
-    set(BLASFEO_INSTALL_DIR "/opt/blasfeo" CACHE PATH "Path to blasfeo installation directory")
-    # find_library(blasfeo NAMES blasfeo PATHS ${BLASFEO_INSTALL_DIR}/lib)
-
-    find_path(BLASFEO_INCLUDE_DIR NAMES "blasfeo.h" PATHS ${BLASFEO_INSTALL_DIR}/include)
-    find_library(BLASFEO_LIBRARY NAMES blasfeo HINTS ${BLASFEO_INSTALL_DIR}/lib)
-
-    # Create an imported static target for blasfeo
-    add_library(blasfeo STATIC IMPORTED GLOBAL)
-    # Set the path to the library file
-    set_target_properties(blasfeo PROPERTIES IMPORTED_LOCATION ${BLASFEO_LIBRARY})
-
-    # Set the path to the include directory
-    set_target_properties(blasfeo PROPERTIES INTERFACE_INCLUDE_DIRECTORIES ${BLASFEO_INCLUDE_DIR})
+    find_package(blasfeo REQUIRED)
 
 endif()
 
